content/zh/docs/faq/telemetry.md

```diff
@@ -1,24 +1,90 @@
 ---
 title: "Telemetry in KubeSphere"
-keywords: "Installer, Telemetry, KubeSphere, KubeSphere"
-description: "Telemetry collects aggregate information of the KubeSphere installation."
+keywords: "Installer, Telemetry, KubeSphere, Kubernetes"
+description: "Telemetry collects aggregate information of KubeSphere installation."
 
 Weight: 7100
 ---
 
-Telemetry collects aggregate information about the size of KubeSphere Cluster installations, KubeSphere and Kubernetes versions, Cluster size, the running time of cluster, error logs, etc. We promise that this information is only used by KubeSphere Community to improve the product and we will not shared with any third-parties.
+Telemetry collects aggregate information about the size of KubeSphere clusters installed, KubeSphere and Kubernetes versions, components enabled, cluster running time, error logs, etc. KubeSphere promises that the information is only used by the KubeSphere community to improve products and will not be shared with any third parties.
 
-## What Information will be collected
+## What Information Is Collected
 
-- External Network IP
-- Download Date
-- Kubernetes Version
-- KubeSphere Version
-- Kubernetes Cluster Size
-- The type of operating system
-- Error logs from Installer
-- Components you enabled
-- The running time of Kubernetes Cluster
-- The running time of KubeSphere Cluster
+- External network IP
+- Download date
+- Kubernetes version
+- KubeSphere version
+- Kubernetes cluster size
+- The type of the operating system
+- Installer error logs
+- Components enabled
+- The running time of Kubernetes clusters
+- The running time of KubeSphere clusters
 - Cluster ID
 - Machine ID
+
+## Disable Telemetry
+
+Telemetry is enabled by default when you install KubeSphere, while you also have the option to disable it either before or after the installation.
+
+### Disable Telemetry before Installation
+
+When you install KubeSphere on existing Kubernetes clusters, you need to download the file [cluster-configuration.yaml](https://raw.githubusercontent.com/kubesphere/ks-installer/v3.0.0/deploy/cluster-configuration.yaml) for cluster setting. If you want to disable Telemetry, do not use `kubectl apply -f` directly for this file.
+
+{{< notice note >}} 
+
+If you install KubeSphere on Linux, see [Disable Telemetry after Installation](../telemetry/#disable-telemetry-after-installation) directly.
```

Comment:
Please update the related information in `/zh`.

Reply:
Same above